src/post_section/post_section.jsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import './post_section.css';
+class PostSection extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        let links2 = [
```

Comment:
1. why do you use `let` and not `const`?
2. I think I've already explained that you shouldn't use numbers when you name your variables, find a meaningful name
3. even if you use numbers in names, why is `links2` declared before `links1`?
4. you can declare these variables outside of the component, no need to create them every time the component is rendered